activemodel/lib/active_model/attributes.rb

```diff
@@ -71,12 +71,6 @@ def initialize(*)
     private
 
       def write_attribute(attr_name, value)
-        name = if self.class.attribute_alias?(attr_name)
-          self.class.attribute_alias(attr_name).to_s
-        else
-          attr_name.to_s
-        end
-
         @attributes.write_from_user(attr_name.to_s, value)
```

Comment:
Looks like it should be `@attributes.write_from_user(name, value)`. Isn't it?

Reply:
Seems correct. I updated. Thanks!